modules/jdbc/src/main/java/org/apache/ignite/internal/ConnectionProperties.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.internal.jdbc;
+package org.apache.ignite.internal;
```

Comment:
I would prefer to follow common patter `org.apache.ignite.internal.<name of the module>` which is `org.apache.ignite.internal.jdbc` for module `jdbc`